core/kazoo_modb-1.0.0/src/kazoo_modb.erl

```diff
@@ -319,3 +320,20 @@ delete_if_orphaned(AccountMODb, 'true') ->
     Succeeded = couch_mgr:db_delete(AccountMODb),
     lager:debug("cleanse orphaned modb ~p... ~p", [AccountMODb,Succeeded]),
     Succeeded.
+
+
+%% @public
+-spec get_range(ne_binary(), gregorian_seconds(), gregorian_seconds()) ->
+                       ne_binaries().
+get_range(AccountId, From, To) ->
+    {{FromYear, FromMonth, _}, _} = calendar:gregorian_seconds_to_datetime(From),
+    {{ToYear, ToMonth, _}, _} = calendar:gregorian_seconds_to_datetime(To),
+    Seq = crossbar_util:generate_year_month_sequence({FromYear, FromMonth}
```

Comment:
Check if anything else in Crossbar uses this util function. If not, we should move it here too. Core libs should not be calling application modules!